inventory/debian.cf

```diff
@@ -23,6 +23,14 @@ bundle common inventory_debian
       "mint_release" string  => "$(mint_info[RELEASE][1])" ;
       "mint_codename" string => "$(mint_info[CODENAME][1])" ;
 
+    any::
+      "proc_routes" data => data_readstringarrayidx("/proc/net/route",
+                                                    "#[^\n]*","\s+",20,2048);
```

Comment:
Seems to work directly as well (3.6.6).
Another thing is if you'd prefer bigger maxent / size values?